<launch>

    <arg name="save_path" default="$(find localization)/results/Result_2/icp2_odomM_1123.csv" />
    <param name="use_sim_time" value="true" />

    <node pkg="rviz" type="rviz" name="display_result" output="screen" args="-d $(find localization)/config/nuscenes.rviz" />

    <!-- <node name="map_publisher" pkg="map_tile_loader" type="map_publisher" output="screen">
        <param name="map_path" type="string" value="/home/gogochiou/SDC_ws/data/nuscenes_maps" />
    </node> -->

    <!-- <node pkg="localization" type="pub_map" name="pub_map" output="screen" >
		<param name="map_path" value="/home/gogochiou/SDC_ws/data/nuscenes_maps/nuscenes_map.pcd" />
	</node> -->

    <!-- <node pkg="localization" type="localizer_icp_2_imu" name="localizer_icp_2_imu" output="screen">
        <rosparam file="$(find localization)/config/nuscenes.yaml" command="load" />
        <param name="map_path" type="string" value="/home/gogochiou/SDC_ws/data/nuscenes_map.pcd" />
        <param name="init_yaw" type="double" value="4.734"/>
        <rosparam param="result_save_path" subst_value="True">$(arg save_path)</rosparam>
    </node> -->

    <node pkg="localization" type="localizer_icp_2" name="localizer_icp_2" output="screen">
        <rosparam file="$(find localization)/config/nuscenes.yaml" command="load" />
        <param name="map_path" type="string" value="/home/gogochiou/SDC_ws/data/nuscenes_map.pcd" />
        <param name="init_yaw" type="double" value="4.734"/>
        <rosparam param="result_save_path" subst_value="True">$(arg save_path)</rosparam>
    </node>

</launch>
